htdocs/accountancy/admin/defaultaccounts.php

```diff
@@ -195,7 +195,8 @@
 	print '</td>';
 	// Value
 	print '<td class="right">'; // Do not force class=right, or it align also the content of the select box
-	print $formaccounting->select_account($conf->global->$key, $key, 1, '', 1, 1, 'minwidth100 maxwidth300 maxwidthonsmartphone', 'accountsmain');
+	$key_value = getDolGlobalString($conf->global->$key, $conf->global->$key);
```

Comment:
`$key_value = getDolGlobalString($conf->global->$key);`
2nd parameter to pass optional default value

Reply:
Thanks Frederic, it's resolved